### DISCUSSION TITLE
How to set up Weblate for a New Component and Mapping it to Version Control System?

### DISCUSSION THREAD

**Discussion by vanlinh2204** [WeblateOrg/weblate#9495] [Q&A]
Hello everyone,

I have the component ready in Weblate, I want to connect it to my Version Control System. What are the steps involved in mapping the component to the VCS? Are there any specific configurations or settings I need to be aware of?

Thank you in advance for your assistance!

**Comment by nijel** [comment]
Choose desired integration level and configure the component accordingly: https://docs.weblate.org/en/latest/admin/continuous.html#pushing-changes-from-weblate
